projects/packages/forms/src/contact-form/js/accessible-form.js

```diff
@@ -47,6 +49,17 @@ const initAllForms = () => {
 		.forEach( initForm );
 };
 
+function isFormEmpty( form ) {
+	const clonedForm = form.cloneNode( true );
+	Array.from( clonedForm.querySelectorAll( 'input[type="hidden"]' ) ).forEach( input =>
+		input.remove()
+	);
+	const formData = new FormData( clonedForm );
+	return ! Array.from( formData.values() ).some( value =>
+		value instanceof File ? !! value.size : !! value.trim?.()
```

Comment:
I don't think there is a file input right now, but it should be good to have the check in advance. I'm not sure if more types of values can be inside the form.

Reply:
I think this is a good check to have just in case.